website/blog/2025-04-20-new-minc-extension-inner-loop.md

```diff
@@ -0,0 +1,172 @@
+---
+title: 'MINC: Speed Up Local K8s Dev'
+description: Speed up your development inner loop using a local Kubernetes cluster powered by MicroShift in a container.
+slug: iterate-quickly-inner-loop-with-a-kubernetes-cluster
+authors: [benoitf]
+tags: [podman-desktop, podman, images, kubernetes, microshift]
+hide_table_of_contents: false
+image: /img/blog/iterate-quickly-inner-loop-with-a-kubernetes-cluster/iterate-quickly-inner-loop-with-a-kubernetes-cluster.png
+---
+
+Developers working locally often seek a fast inner loop for coding, building, and testing their OCI images and Kubernetes applications.
+
+For instance, when testing a new image in a Kubernetes pod, the image must be accessible either by pushing it to a private or public registry or by ensuring it's available on the Kubernetes cluster nodes. This often requires additional commands such as `kind load docker-image`, `minikube cache add <image>`, or publishing the image to a third-party registry.
+
+In this blog post, we’ll introduce a new extension called `MINC`, which provides a MicroShift cluster running within an existing Podman Machine environment.
```

Comment:
Better to link `MicroShift` project (https://github.com/openshift/microshift) here.

Reply:
added